fapolicy_analyzer/ui/ancillary_trust_database_admin.py

```diff
@@ -76,47 +76,48 @@ def delete_trusted_files(self, *files):
             changeset.del_trust(file)
         self.__apply_changeset(changeset)
 
-    def on_trust_selection_changed(self, trust):
-        self.selectedFile = trust.path if trust else None
+    def on_trust_selection_changed(self, trusts):
+        self.selectedFile = trusts if trusts else None
         trustBtn = self.get_object("trustBtn")
         untrustBtn = self.get_object("untrustBtn")
-        if trust:
-            status = getattr(trust, "status", "").lower()
-            trusted = status == "t"
-            trustBtn.set_sensitive(not trusted)
-            untrustBtn.set_sensitive(trusted)
-
-            if isinstance(trust, Trust):
-                self.trustFileDetails.set_in_database_view(
-                    f(
-                        _(
-                            """File: {trust.path}
+        for trust in trusts:
+            if trust:
```

Comment:
I'm not sure why you need to loop here.  We can only display 1 file at a time in the details, so you should just pull either the first or last one off the list to display to make this more efficient.